accepted/00101-make_cve_reports_possible.md

```diff
@@ -0,0 +1,60 @@
+- Feature Name: make_cve_reports_possible
+- Start Date: 2024-04-12 
+
+# Summary
+[summary]: #summary
+
+Create a meaninful CVE Audit that would ran as a report and it would be sorted per CVE-Number
+
+# Motivation
+[motivation]: #motivation
+
+- Why are we doing this? The CVE Audit is static and does not provide an overview from the hosts affected by a CVE.
+- What use cases does it support? Produce real-time reports sorted by CVE-Number for the auditors and security teams
+- What is the expected outcome? Have a table with the CVE Numbers and what hosts are affected. Have a report that could be called via REST-API and also could be integrated on every CI/CD workflows from operational teams.
+
+We have a static and not a understable overview from the hosts affected by a CVE. As today, one should open the CVE Audit option on the menu, search for a CVE number, add to the form and expect a list from hosts affected by only this CVE. If there is a higher number from CVEs would be a lot of working doing a manual search per CVE Number. This makes the life from infrastructure managers so difficult -it is not possible to have an overview that is always updated and showed in the dashboard -
+and it is also not possible to have a generated report per PDF from the hosts affected by a CVE(a report sorted per CVE-Number. It should be also provided a way to generate this reports per REST-API and make it possible to have it integrated in a CI/CD Workflow.
+
+# Detailed design
+[design]: #detailed-design
+
+Steps for the implementation:
+- Create a stored procedure/function on the reporting DB making it call on demand by the frontend/backend code.
```

Comment:
We try to avoid store procedures as much as possible. The reason is because they are hard to maintain and debug.
Also, the report DB and "main" DB are separated in different schemas and can also be on different machines in the future.

Reply:
ok :)

Reply:
I would check the other RFC and I think that we can combine the work here.